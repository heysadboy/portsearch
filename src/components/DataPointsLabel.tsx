import { FC } from "react";

interface IDataPointLabelProps {
    info: string
}

//Information regarding number of Data Points
const DataPointLabel: FC<IDataPointLabelProps> = ({ info }) => {
    return (<div className="ui orange label data-point-container">{info}</div>);
};

export default DataPointLabel;