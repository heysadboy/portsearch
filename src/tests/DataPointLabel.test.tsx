import TestRenderer from "react-test-renderer";
import DataPointsLabel from "../components/DataPointsLabel";
 
test("DataPointsLabel Component Testing.", () => {
    const component = TestRenderer.create(
        <DataPointsLabel info="10 Data Points" />
    );

    let tree: any = component.toJSON();
    expect(tree).toMatchSnapshot();

    //Checking if label is correct
    expect(component.root.findByType("div").props.children).toEqual("10 Data Points");
});