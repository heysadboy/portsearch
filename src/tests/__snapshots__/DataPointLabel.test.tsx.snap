// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataPointsLabel Component Testing. 1`] = `
<div
  className="ui orange label data-point-container"
>
  10 Data Points
</div>
`;
